realm/realm-library/src/objectServer/java/io/realm/mongodb/mongo/MongoClient.java

```diff
@@ -18,21 +18,25 @@
 
 import org.bson.codecs.configuration.CodecRegistry;
 
-import io.realm.mongodb.User;
 import io.realm.internal.Util;
+import io.realm.internal.common.TaskDispatcher;
 import io.realm.internal.objectstore.OsMongoClient;
 
 /**
  * The remote MongoClient used for working with data in MongoDB remotely via Realm.
  */
 abstract public class MongoClient {
```

Comment:
I recall adding getters for `App` and `User` on `Functions` for convenience. But can't recall the use case...maybe something like `functions.getApp().getConfiguration().getDefaultCodecRegistry()`. 

Maybe we should consider adding these for consistency. @cmelchior can you remember the use case?